bot/buttons.py

```diff
@@ -92,6 +98,11 @@ async def set_main_menu(bot:Bot):
         BotCommand(command='/help',description= 'Справка по работе бота'),
         BotCommand(command='/about_us',description='Команда, которая расскажет тебе про наши отделы'),
         BotCommand(command='/quest',description= 'Пройди квест для определения своего отдела'),
+
+        #удалить при надобности
+        BotCommand(command='/back', description='Возвращает в начало квеста'),
+        BotCommand(command='/exit', description='Выходит из состояния квеста '),
```

Comment:
@NikitaZavorin Is it possible to show these commands only when user is in question section?

Reply:
I'm not sure, but I can try. Or you can add some transparent hints for the user. I have attached an example

![image](https://github.com/user-attachments/assets/b8b4e766-4a73-443f-919e-a2ca91c58b7c)

Reply:
This was fixed in [this resolved conversation](https://github.com/wak3upalex/eestecbot/pull/35#discussion_r1780105416)